tests/functionaltests/suites/oarec/test_oarec_functional.py

```diff
@@ -228,6 +228,12 @@ def test_items(config):
     assert content['numberReturned'] == 10
     assert content['features'][5]['properties']['title'] == 'Lorem ipsum dolor sit amet'  # noqa
 
+    params = {'SoRtBy': '-title'}
```

Comment:
typo?

Reply:
This is an added test to verify that case-insensitive query parameter names are supported.